apps/docs/src/app/core/component-docs/action-bar/action-bar-docs.module.ts

```diff
@@ -15,6 +15,7 @@ import { API_FILES } from '../../api-files';
 import { SharedDocumentationModule } from '../../../documentation/shared-documentation.module';
 import { ActionBarModule } from '@fundamental-ngx/core';
```

Comment:
What about documentation page, do you have the sample in github what text do we have there? I think since you have it opened we can update this part too. What do you think?

Reply:
Shall i take this documentation as a separate Task?